### ISSUE TITLE
Revamp navigation/page anchors

### ISSUE THREAD

**Issue by billtomczak** [OSTraining/OSCampus#158]
1. pathways view should only show selected pathways
2. Courses view to provide anchor/single url for all courses (classes)
3. Remove breadcrumbs
4. Fix redirections for old guru urls.
5. Fix references in db if needed

**Comment by billtomczak** [comment]
Addressed by dc6b036f1da43b86c4a365628534e07854c1dde8...bde2a96039961aae334ea952c1da13ea326ae67b

Also updated htacces to the following:

```
# Guru to OSCampus redirections
RewriteRule ^courses/session/[^/]+/([^/]+)/[^/]+/(.*)$ class/$1/$2 [L,R=301]
RewriteRule ^courses/categories/([^/]+)/(.*)$ classes/$1/$2 [L,R=301]
RewriteRule ^courses/class/[^/]+/(.*)$ class/$1? [L,R=301]

# OSCampus URL changes
RewriteRule ^classes/[^/]+/(.+)$ class/$1 [L,R=301]
```

**Comment by htmgarcia** [comment]
I love those regex examples!
I didn't thought before a redirection of a redirection could be possible (about # OSCampus URL changes)
